fw/http_frame.c

```diff
@@ -600,16 +616,17 @@ __tfw_h2_stream_unlink(TfwH2Ctx *ctx, TfwStream *stream)
 
 	if (hmreq) {
 		hmreq->stream = NULL;
+
 		/*
-		 * If the request is linked with a stream, but not complete yet,
-		 * it must be deleted right here to avoid leakage, because in
-		 * this case it is not used anywhere yet. When request is
-		 * assembled and complete, it will be removed (due to some
-		 * processing error) in @tfw_http_req_process(), or in other
-		 * cases controlled by server connection side (after adding to
-		 * @fwd_queue): successful response sending, eviction etc.
-		 */
-		if (!test_bit(TFW_HTTP_B_FULLY_PARSED, hmreq->flags))
+		* If the request is linked with a stream, but not complete yet,
+		* it must be deleted right here to avoid leakage, because in
+		* this case it is not used anywhere yet. When request is
+		* assembled and complete, it will be removed (due to some
+		* processing error) in @tfw_http_req_process(), or in other
+		* cases controlled by server connection side (after adding to
+		* @fwd_queue): successful response sending, eviction etc.
+		*/
```

Comment:
Comments are the same, just a broken identation and extra empty line - the change doesn't make much sense

Reply:
Fixed